src/v/kafka/protocol/list_partition_reassignments.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "kafka/protocol/schemata/list_partition_reassignments_request.h"
```

Comment:
please split the kafka changes into a separate commit sequenced after the other changes (e.g. to cluster)